### ISSUE TITLE
OpenstreetCam open source their data and code

### ISSUE THREAD

**Issue by hungrymonkey** [curbmap/curbmap-ml#2]
https://www.openstreetmap.org/user/mvexel/diary/43942

**Comment by eselkin** [comment]
Yeah. I saw that in their listserv. But this repo is not about that. :-)... However, OpenStreetCam has such limited data.
